apiserver/worker/compiler.py

```diff
@@ -680,63 +680,139 @@ def get_run_lang(submission_dir):
                     if line[0] == '#':
                         return line[1:-1]
 
-def compile_anything(bot_dir, installTimeLimit=600, timelimit=600, max_error_len = 3072):
+
+INSTALL_ERROR_START = "Possible errors running install.sh. stdout output as follows:"
+INSTALL_ERROR_MID = "End of install.sh stdout. stderr as follows:"
+INSTALL_ERROR_END = "End of install.sh stdout."
```

Comment:
INSTALL_ERROR_END isn't put at the end of the install.sh stdout so instead should probably be something like "End of install.sh output."